cf_xarray/accessor.py

```diff
@@ -312,7 +354,12 @@ def _get_all(obj: Union[DataArray, Dataset], key: str) -> List[str]:
     One or more of ('X', 'Y', 'Z', 'T', 'longitude', 'latitude', 'vertical', 'time',
     'area', 'volume'), or arbitrary measures, or standard names
     """
-    all_mappers = (_get_axis_coord, _get_measure, _get_with_standard_name)
+    all_mappers = (
+        _get_custom_criteria,
+        _get_axis_coord,
```

Comment:
I kept `_get_axis_coord` separate for now to keep test passing. We can do some refactoring and cleanup once we get all the logic working.

Reply:
Sounds good.